From c5819f037769882112f285dee8579f28f5350fcc Mon Sep 17 00:00:00 2001
From: Timo Herbrecher <t.herbrecher@gateware.de>
Date: Fri, 5 Jul 2019 10:52:53 +0200
Subject: [PATCH 5/7] ls1088a: Use HWA clock for esdhc

To support high speed transfers on esdhc interface as HS200 for eMMC the
platform clock could not be used as reference. Use HWA clock instead.

Signed-off-by: Timo Herbrecher <t.herbrecher@gateware.de>
---
 arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
index 536caf7d9bd4..827b6eb19989 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
@@ -375,7 +375,7 @@
 			compatible = "fsl,ls1088a-esdhc", "fsl,esdhc";
 			reg = <0x0 0x2140000 0x0 0x10000>;
 			interrupts = <0 28 0x4>; /* Level high type */
-			clock-frequency = <0>;
+			clocks = <&clockgen 2 1>;
 			voltage-ranges = <1800 1800 3300 3300>;
 			sdhci,auto-cmd12;
 			little-endian;
-- 
2.17.1

